opus/application/apps/tools/file_utils.py

```diff
@@ -237,3 +237,24 @@ def get_pds_preview_images(opus_id_list, preview_jsons, sizes=None,
         image_list.append(data)
 
     return image_list
+
+def get_displayed_browse_products(opus_id, version_name='Current'):
+    """Given an opus_ids, return a list of browse products urls displayed
+       in detail tab.
+    """
+    browse_products = get_pds_products(opus_id_list=opus_id,
+                                       product_types=settings.DISPLAYED_BROWSE_PRODUCTS)
+    selected_browse_products = browse_products[opus_id].get(version_name, [])
+    # When there is no preview image, we return settings.THUMBNAIL_NOT_FOUND
+    if len(selected_browse_products) == 0:
+        return [(settings.THUMBNAIL_NOT_FOUND, settings.THUMBNAIL_NOT_FOUND)]
+    res = []
+    # One opus id could have multiple previews, for example:
+    # co-rss-occ-2008-039-rev058c-x43-i
+    # co-uvis-occ-2005-232-alpsco-i
+    for p in selected_browse_products:
+        for browse_med_url in selected_browse_products[p]:
+            browse_full_url = browse_med_url.replace('_med.', '_full.')
```

Comment:
This really isn't the right way to do this, because there's no guarantee that preview/browse filenames will follow this format. We should be using the PdsViewable capability somehow, but I don't think it currently supports what we want to do and would require changes to the import pipeline. You should submit a new GitHub issue after this pull is merged saying this eventually needs to be fixed.